backend/src/v5/models/invitations.js

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const Invitations = {};
+const db = require('../handler/db');
+
+const COL_NAME = 'invitations';
+
+const findMany = (query, projection, sort) => db.find('admin', COL_NAME, query, projection, sort);
+
+Invitations.getInvitationsByTeamspace = (teamspace) => findMany({ 'teamSpaces.teamspace': teamspace });
```

Comment:
shoudl provide a projection. invitations can be massive, we don't want to pull every (esp if all we're doing is counting)